bound/kt/src/test/kotlin/web5/sdk/dids/BearerDidTest.kt

```diff
@@ -0,0 +1,22 @@
+package web5.sdk.dids
+
+import org.junit.jupiter.api.Test
+import web5.sdk.crypto.keys.InMemoryKeyManager
+import web5.sdk.dids.methods.jwk.DidJwk
+import web5.sdk.rust.ed25519GeneratorGenerate
+
+class BearerDidTest {
+
+    @Test
+    fun `test basic bearer did creation`() {
+        val privateJwk = ed25519GeneratorGenerate()
+
+        val keyManager = InMemoryKeyManager(listOf())
+        val publicJwk = keyManager.importPrivateJwk(privateJwk)
+
+        val didJwk = DidJwk(publicJwk)
+
+        // TODO: This is throwing an error on the rust side
```

Comment:
```
web5.sdk.rust.RustCoreException
    at web5.sdk.rust.FfiConverterTypeRustCoreError.lift(web5.kt:4910)
    at web5.sdk.rust.FfiConverterTypeRustCoreError.read(web5.kt:4916)
    at web5.sdk.rust.RustCoreException$ErrorHandler.lift(web5.kt:4897)
    at web5.sdk.rust.RustCoreException$ErrorHandler.lift(web5.kt:4890)
    at web5.sdk.rust.Web5Kt.uniffiCheckCallStatus(web5.kt:270)
    at web5.sdk.rust.Web5Kt.access$uniffiCheckCallStatus(web5.kt:1)
    at web5.sdk.rust.BearerDid.<init>(web5.kt:7018)
    at web5.sdk.dids.BearerDid.<init>(BearerDid.kt:34)
    at web5.sdk.dids.BearerDidTest.test basic bearer did creation(BearerDidTest.kt:20)
    at java.base/java.lang.reflect.Method.invoke(Method.java:580)
    at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
    at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
```
    
    deep error is happening, I'm investigating, but I think it is something on the rust side

Reply:
resolved! thanks @KendallWeihe !